kratos/python/add_containers_to_python.cpp

```diff
@@ -644,6 +644,11 @@ void  AddContainersToPython(pybind11::module& m)
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, INTEGRATION_COORDINATES )
 
 
+    //for ULF (surface_tension) application:
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,TRIPLE_POINT)
```

Comment:
Why do you add these variables to the Core?

They are only used in your application from what I can see

Reply:
@playpaolo this was also changed at some point